.github/workflows/merge-queue.yml

```
@@ -13,193 +18,147 @@
+      TEST_CHUNKS: ${{ steps.build_info.outputs.test_chunks }}
+      E2E_CHUNKS: ${{ steps.build_info.outputs.e2e_chunks }}
+      E2E_BUILD_ID: ${{ steps.build_info.outputs.build_iD }}
+      BUILD_CHUNKS: ${{ steps.build_info.outputs.build_chunks }}
+      NX_BASE: ${{ steps.build_info.outputs.nx_base }}
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - uses: ./.github/actions/prep-deps
```

actionlint (1.7.4)

missing input "cypress_cache_folder" which is required by action "Prep Dependencies" defined at "./.github/actions/prep-deps". all required inputs are "cypress_cache_folder", "generated_files"

(action)